capa/main.py

```diff
@@ -518,6 +518,7 @@ def get_extractor(
 
         import capa.features.extractors.smda.extractor
 
+        logger.warning("Deprecation warning: v4.0 will be the last capa version to support the SMDA backend.")
```

Comment:
this logging strategy is good. we may also want to consider using the `warnings` module (https://docs.python.org/3/library/warnings.html) with a `DeprecationWarning` instance.

https://docs.python.org/3/library/exceptions.html#DeprecationWarning

Reply:
added in f4d0e8d